src/App.js

```diff
@@ -1,56 +1,56 @@
+// Stars Align App
```

Comment:
Could consider adding this in a text file so it doesn't add a lot of lines to the js file?